src/main/scala/org/deepdive/ddlog/DeepDiveLogParser.scala

```diff
@@ -252,7 +252,7 @@ class DeepDiveLogParser extends JavaTokenParsers {
 
   def relationType: Parser[RelationType] =
     ( "like" ~> relationName ^^ { RelationTypeAlias(_) }
-    | rep1sep(columnDeclaration, ",") ^^ {
+    | "(" ~> rep1sep(columnDeclaration, ",") <~ ")" ^^ {
```

Comment:
It seems to me `like R` itself is enough to express the expected schema is the same as the schema of `R`, and `rows` is kind of redundant here...